### PR TITLE
improve readme, remove cli, remove test requiring .each

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
remove cli feature 
removed test requiring them to use ranges, convert range to array, use .each (they haven't learned any of that yet)

included discussion of ternary operator in readme

made separate PR for new solution branch

@jmburges @aviflombaum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,74 @@
----
-tags: ternary, ||=, operators
-language: ruby
-resources: 4
----
-
 # Operations
```
Comment (jmburges) [inline README.md]: I would rename this to "Operators" not "Operations"

File: README.md [hunk 2]
```diff
@@ -1,38 +1,74 @@
...
 
 ## Objectives
 
 Get familiar with: 
 
+* Learn about the ternary operator
+* Review using comparison operators and boolean operators
 * passing arguments to methods
```
Comment (jmburges) [inline README.md]: hmmm the point of a lab on operators also has the objective of passing arguments to methods?

File: README.md [hunk 3]
```diff
@@ -1,38 +1,74 @@
...
+How does this work? In the above statement, the code before the `?` is evaluated as a boolean expression. If it returns true, the code on the left side of the `:` will run, otherwise the code on the right of the `:` will run. 
+
+Think about it like this: 
+
+```ruby
+boolean_expression ? I_happen_if_boolean_expression_is_true : I_happen_if_boolean_is_false
+```
+## Why Use the Ternary Operator?
+
+The ternary operator has a few things going for it. First of all, as Rubyists, we like to think about design. Is our code needlessly complex? Are our methods too long? Is our code self-explanatory––i.e., can someone reading it easily understand what it does? These are good questions to keep in mind as you learn to build more and more complicated programs. In fact, Ruby is a language that lends itself particularly well elegant design. There are many design patterns that you'll learn about later on in this course, but for now, just keep the above questions in mind.
```
Comment (jmburges) [inline README.md]: Leet's give a more concrete opinion on when to use the ternary operator. In my opinion it should never be used....but I know many people disagree with me. How about only if the condition is `incredibly simple` or something?

File: README.md [hunk 4]
```diff
@@ -1,38 +1,74 @@
...
+	* Use an `if/else` block to build the `unsafe?`. It returns `true` if the speed is below 40 or above 60. 
   * Going 30 mph on the freeway would be unsafe, as would going 95 mph. 
   * Going 50 miles per hour, however, would return `false` as that's within the "safe" range.
-* Build the method `not_safe?` that is a version of your previous `unsafe?` method but with a [ternary operator](http://www.codecademy.com/glossary/ruby/ternary-operator) instead of an if/else block.
-* Build a method named `sum_machine` that takes two parameters and sums them, along with every integer that falls between them, together. Incoporate the `+=` [shorthand assignment operator](http://www.techotopia.com/index.php/Ruby_Operators#Ruby_Assignment_Operators).
-  * For instance, `sum_machine(3, 7)` should return `25` (3 + 4 + 5 + 6 + 7) while sum_machine(5, 1) should return `15` (1 + 2 + 3 + 4 + 5).
+4.  Build the method `not_safe?` that is a version of your previous `unsafe?` method but with a [ternary operator](http://www.codecademy.com/glossary/ruby/ternary-operator) instead of an `if/else block`.
```
Comment (jmburges) [inline README.md]: kinda trying to avoid the outside links...Let's keep it all in house here

File: README.md [hunk 5]
```diff
@@ -1,38 +1,74 @@
...
 
-You can interact with your methods by typing `ruby bin/runner.rb` into your terminal from the root of this directory. This will run the file `runner.rb`. This file calls on the `call` method in `lib/cli.rb`. In another lab, you'll be building a CLI so familarize yourself with the way they work by looking at the code in `lib/cli.rb`. A good place to start would be `def call`.
+**A Debugging Reminder:** You can interact with your methods by typing `require 'pry'` at the top of your file and then placing the line `binding.pry` inside the method that you want to play around with. Then, when you run the test suite using `rspec` or `learn` in your terminal, your program will freeze when it hits the binding and you'll be able to explore your method inside your console.
```
Comment (jmburges) [inline README.md]: :tada:

**Comment by jmburges** [comment]
Which [issues](https://github.com/learn-co-curriculum/operations/issues) doe this resolve?

**Comment by SophieDeBenedetto** [comment]
latest commit fixes all issues


On Thu, Jun 25, 2015 at 2:22 PM, Joe Burgess <notifications@github.com>
wrote:

> Which issues <https://github.com/learn-co-curriculum/operations/issues>
> doe this resolve?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learn-co-curriculum/operations/pull/8#issuecomment-115353376>
> .
>



-- 
Sophie DeBenedetto

File: README.md (hunk 3 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed w latest commit.

**Pull request merged**